python/langsmith/evaluation/integrations/_base.py

```diff
@@ -0,0 +1,11 @@
+from typing import Protocol, runtime_checkable
+
+from langsmith.evaluation.evaluator import RunEvaluator
+
+
+@runtime_checkable
+class EvaluatorWrapper(Protocol):
+    """A protocol that represents an evaluator wrapper."""
+
+    def as_run_evaluator(self) -> RunEvaluator:
```

Comment:
I'm somewhat ambivalent on whether we want this as a constructor or just to directly subclass. Maybe simpler to do the latter